README.md

```diff
@@ -21,12 +21,13 @@ development build and reporting any issues you run into.
 > Prebuilt images for cert-manager are made available on Dockerhub.
 
 This guide sets up cert-manager to run as a Deployment on your Kubernetes
-cluster. It will then go on to set up the Letsencrypt staging server as a
-Certificate issuer, and request a Certificate for a domain you control using
-the HTTP01 challenge mechanism.
+cluster.
+It will then go on to set up the [Let's Encrypt ACME staging server](https://letsencrypt.org/docs/staging-environment/)
+as a Certificate issuer, and request a Certificate for a domain you control
+using both the HTTP01 and DNS01 challenge mechanisms.
 
-This guide will configure cert-manager to attempt to fulfill Certificate
-resources in all namespace in your cluster.
+By default, it will be configured to fulfil Certificate resources in all
+namespaces.
 
 ### 0. Pre-requisites
```

Comment:
❓ This gets rendered as item 0. Is that intended?

Reply:
Yep, pre-req is kind of before everything else.